pkgs/development/python-modules/qgrid/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchpatch
+, fetchPypi
+, ipywidgets
+, notebook
+, pandas
+}:
+
+buildPythonPackage rec {
+  pname = "qgrid";
+  version = "1.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-/or1tQgzCE3AtqJlzRrHuDfAPA+FIRUBY1YNzneNcRw=";
+  };
+
+  propagatedBuildInputs = [
+    ipywidgets
+    notebook
+    pandas
+  ];
+
+  patches = [
+    # Fixes compatibility of qgrid with ipywidgets >= 8.0.0
+    # See https://github.com/quantopian/qgrid/pull/331
+    (fetchpatch {
+      url = "https://github.com/quantopian/qgrid/pull/331/commits/8cc50d5117d4208a9dd672418021c59898e2d1b2.patch";
+      hash = "sha256-+NLz4yBUGUXKyukPVE4PehenPzjnfljR5RAX7CEtpV4=";
+    })
+  ];
+
+  doCheck = false;
```

Comment:
This needs a comment.
Why not run https://github.com/quantopian/qgrid/tree/master/qgrid/tests ?

Reply:
I did not find an easy way to skip a specific test with the `unittest` backend.
Hence, I disabled them completely.
I added a comment.

Reply:
Can't we use pytestCheckHook instead?

Reply:
Indeed ! I didn't know we could do that.
I was able to ignore the two failing tests.